src/main/java/org/folio/bulkops/service/InstanceReferenceService.java

```diff
@@ -144,4 +160,41 @@ public String getNoteTypeIdByName(String name) {
   public List<InstanceNoteType> getAllInstanceNoteTypes() {
     return instanceNoteTypesClient.getInstanceNoteTypes(Integer.MAX_VALUE).getInstanceNoteTypes();
   }
+
+  @Cacheable(cacheNames = "sortedInstanceNoteTypes")
+  public List<String> getSortedInstanceNoteTypes() {
```

Comment:
Hi @siarhei-charniak , for caching logic it is necessary to have unique identifier per tenant like tenantId as argument.

Reply:
Hi @alekGbuz , good catch, thanks! Removing this method since it is not used.